src/main/java/com/puppycrawl/tools/checkstyle/site/XdocsTemplateSink.java

```diff
@@ -49,6 +56,40 @@ public class XdocsTemplateSink extends XdocSink {
     public XdocsTemplateSink(Writer writer, String encoding) {
         super(writer);
         this.encoding = encoding;
+        this.writer = new PrintWriter(writer);
+    }
+
+    /**
+     * Customization of org.apache.maven.doxia.sink.impl.XhtmlBaseSink.unifyEOL() method.
+     *
+     * @param text the text to scan.
+     * @return a String that contains only Linux System EOLs.
+     */
+    protected String unifyToLinuxEndOfLine(String text) {
+        return text
+                .replace("\r\n", XdocsTemplateSink.LINUX_EOL)
+                .replace("\r", XdocsTemplateSink.LINUX_EOL);
```

Comment:
can we avoid class name usage before LINUX_EOL ?

Reply:
`LINUX_EOL` is static so either `this` or class name could be used. Otherwise I have to make the methods static.
So decide what you want me to proceed with?

Reply:
Done